include/triton/Dialect/TritonGPU/IR/TritonGPUAttrDefs.td

```diff
@@ -1002,11 +1002,11 @@ An encoding for tensors that have been produced by MFMA matrix core instructions
 available on AMD Instinct GPUs of CDNA architectures.
 
 It is characterized by the following parameters:
-- `versionMajor` and `versionMinor` indicates the GPU architecture:
-  - 1.0: gfx908, i.e. CDNA1
-  - 2.0: gfx90a: i.e. CDNA2
-  - 3.0: gfx942: CDNA3
-  - 4.0: gfx950: CDNA4
+- `version` indicates the GPU architecture:
+  - 1: gfx908, i.e. CDNA1
+  - 2: gfx90a: i.e. CDNA2
```

Comment:
Nit: Can you also drop the "i.e." to be consistent with 3/4?

Reply:
done